api/app/signals/apps/email_integrations/templates/email/signal_created.txt

```diff
@@ -1,37 +1,22 @@
-Bedankt!
-
-Uw melding ({{ signal.id }}):
-
-{{ signal.text }}
+Bedankt voor uw melding
+Meldingnummer {{ signal.sia_id }}
 
 Wat doen we met uw melding?
 
 {{ afhandelings_text }}
 
-*** Details van uw melding ***
-
-{% if signal.location.address_text %}
-Hier is het : {{ signal.location.address_text }}
-{% endif %}
+Informatie over uw melding
 
-Tijdstip : {{ signal.incident_date_start|date:"DATETIME_FORMAT" }}
-{% if signal.text_extra %}
+Gemeld op: {{ signal.created_at|date:"DATETIME_FORMAT" }}
+Locatie: {{ signal.location.address_text }}
```

Comment:
niet alle meldingen hebben een adres, dus dit kan fout gaan

Reply:
Added extra check to see if there is a location, if so render it in the template